FhirToDataLake/test/Microsoft.Health.Fhir.Synapse.Core.UnitTests/Microsoft.Health.Fhir.Synapse.Core.UnitTests.csproj

```diff
@@ -25,6 +25,7 @@
   <ItemGroup>
     <ProjectReference Include="..\..\src\Microsoft.Health.Fhir.Synapse.Core\Microsoft.Health.Fhir.Synapse.Core.csproj" />
     <ProjectReference Include="..\..\src\Microsoft.Health.Fhir.Synapse.DataWriter\Microsoft.Health.Fhir.Synapse.DataWriter.csproj" />
+    <ProjectReference Include="..\Microsoft.Health.Fhir.Synapse.DataClient.UnitTests\Microsoft.Health.Fhir.Synapse.DataClient.UnitTests.csproj" />
```

Comment:
Just curious why will one unit test project depend on another unit test project?

Reply:
Because I want to reuse the test data used by data client in in job tests which includes search bundles with next link. Maybe we can extract a common test project that holds all test data later~

Reply:
Got it. It would be better to share a common test data folder instead of adding project references. :smile: